### DISCUSSION TITLE
Use correct term: "Opacity" rather than "Transparency"

### DISCUSSION THREAD

**Discussion by xeruf** [osmandapp/OsmAnd#12455] [General]
In `Configure profile > Configure Map > Terrain`, the slider says "Transparency", even though what the number actually sets is the Opacity, which is the exact opposite.

I wanted to submit a PR right away but couldn't find it in the code.

**Comment by vshcherb** [comment]
Transparency doesn't explain what's exactly transparent and anyway with many languages it will be confusing translation. 
If we talk about 2 layers it's not clear without explanation which layer is opaque and which is transparent (note user doesn't need to know the order of the layers)

**Comment by xeruf** [comment]
> Transparency doesn't explain what's exactly transparent and anyway with many languages it will be confusing translation.
> If we talk about 2 layers it's not clear without explanation which layer is opaque and which is transparent (note user doesn't need to know the order of the layers)

Since the terrain layer is on top (clearly evident when looking at city names, which is a weird ordering btw) it is pretty clear that the slider sets the opacity of that layer. Why close when it isn't solved?

**Comment by vshcherb** [comment]
It is closed because the solution is not clear. I will transform that issue in the discussion to see others people feedback and probably solution will be found.
